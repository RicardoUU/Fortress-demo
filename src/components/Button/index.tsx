import React,{ FC } from 'react';
import * as S from './styled';


const Button:FC = () => (
  <S.Button>
    按钮
  </S.Button>
)

export default Button;