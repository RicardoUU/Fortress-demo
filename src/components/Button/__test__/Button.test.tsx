import React from 'react';
import renderer from 'react-test-renderer';
import { describe, expect, test } from 'vitest';
import Button from '..';

describe('Button', () => {
  test('Button component renders correctly', () => {
    const component = renderer.create(
      <Button />
    )

    const tree = component.toJSON()

    expect(tree).toMatchSnapshot()
  })

  test('The greetee prop works', () => {
    const component = renderer.create(
      <Button />
    )

    const tree = component.toJSON()

    expect(tree).toMatchSnapshot()
  })
})