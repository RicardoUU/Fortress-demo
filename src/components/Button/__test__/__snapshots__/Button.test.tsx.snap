// Vitest Snapshot v1

exports[`Button > Button component renders correctly 1`] = `
<button
  className="sc-bcXHqe ggjowu"
>
  xxxx
</button>
`;

exports[`Button > The greetee prop works 1`] = `
<button
  className="sc-bcXHqe ggjowu"
>
  xxxx
</button>
`;
